### DISCUSSION TITLE
Spotting warnings about Microsoft Naming Convention

### DISCUSSION THREAD

**Discussion by johnsidoli** [abraunegg/onedrive#2079] [Q&A] [answered]
I've noticed (only because I was browsing my log) that I have quite a few log events where files have been skipped due to Microsoft naming conventions... is there a way I can search my files for these and deal with them so that they sync properly?

e.g.

`2022-Aug-05 09:48:18.4101991	Skipping item - invalid name (Microsoft Naming Convention): ./john/docs/summer2022/july/smith/ smith_2022.docx`

(looks like a leading space character on this one)

**Comment by abraunegg** [comment, marked as answer]
@johnsidoli 
> is there a way I can search my files for these and deal with them so that they sync properly?

Unfortunately Google is your friend for this one ... finding files & renaming them so that they are compliant with Microsoft's specification is not something I can assist with.

**Comment by johnsidoli** [comment]
Fair enough, thanks @abraunegg
